javascript/lib/api/src/client/request-factory/request-sender.ts

```diff
@@ -51,6 +51,19 @@ export abstract class RequestSender {
       });
   }
 
+  public fetchMergeRequest<T>(options: ParseRequest<T>): Promise<MergeResponse<T>> {
+    return this.fetchRequest(options).then(response => {
+      if (response.status === 204) {
```

Comment:
As far as I remember a PATCH / merge could also create a new feature or property ..
So IMO checking for the status code 201 is missing here.

Reply:
this should be resolved by using fetchPutRequest()